src/script/page/components/FeatureConfigChange/FeatureConfigChangeNotifier/FeatureConfigChangeNotifier.ts

```diff
@@ -44,15 +43,14 @@ import {loadFeatureConfig, saveFeatureConfig} from './FeatureConfigChangeNotifie
 import {Config} from '../../../../Config';
 import {TeamState} from '../../../../team/TeamState';
 
-const featureNotifications: Partial<
-  Record<
-    FEATURE_KEY,
-    (
-      oldConfig?: Feature<any> | FeatureWithoutConfig,
-      newConfig?: FeatureWithoutConfig | Feature<any> | undefined,
-    ) => undefined | {htmlMessage: string; title: StringIdentifer; primaryAction?: Action}
-  >
-> = {
+type FeatureMessageGenerator = {
+  [K in keyof FeatureList]: (
+    oldConfig?: FeatureList[K],
+    newConfig?: FeatureList[K],
+  ) => undefined | {htmlMessage: string; title: StringIdentifer; primaryAction?: Action};
+};
```

Comment:
This is an improvement of the type, now the `oldConfig` and `newConfig` will be correctly typed with the actual feature config